src/tests/test_sexual_behaviour.py

```diff
@@ -77,3 +79,21 @@ def test_behaviour_updates():
         pop.sexual_behaviour.update_sex_groups(pop.data)
     subsequent_groupings = pop.data["sex_behaviour"]
     assert(any(initial_groupings != subsequent_groupings))
+
+
+def test_initial_sex_behaviour_groups():
```

Comment:
This makes sense but, I think, could be a bit clearer.
I think it'd be very helpful to have a docstring here explaining exactly the aim of the test (that initial distribution matches expected one within a tolerance of ...). Failing that (or even so!), rename some variables or add a comment with their meaning, especially E (even though the symbol matches the expectation - no pun intended).